eng/publish.proj

```diff
@@ -104,4 +104,56 @@
 
     <Message Importance="High" Text="Publishing %(SymbolPackagesToPublish.Identity) to $(SymbolServerPath)"/>
   </Target>
-</Project>
\ No newline at end of file
+
+  <Target Name="PublishTestAssets">
+    <Error Text="The ExpectedFeedUrl property wasn't supplied." Condition="'$(ExpectedFeedUrl)' == ''" />
+    <Error Text="The AccountKey property wasn't supplied." Condition="'$(AccountKey)' == ''" />
+    <Error Text="FilesToPublishPattern property wasn't supplied." Condition="'$(FilesToPublishPattern)' == ''" />
+
+    <PropertyGroup>
+      <_TestAssetVersion>$(PackageVersion)-$(VersionSuffix)</_TestAssetVersion>
+      <_FileRelativePathBase>corefx-tests/$(_TestAssetVersion)/$(OSGroup).$(ArchGroup)/$(TargetGroup)/</_FileRelativePathBase>
```

Comment:
Is TargetGroup needed? Will another repo run corefx tests targeting uap/netfx?

Reply:
It is for the future. Currently none of the uap/netfx jobs publish tests but we may want them in the future and I assume (maybe incorrectly) that they are built differently than netcoreapp framework.